rules/windows/lateral_movement_scheduled_task_powershell_source.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/12/15"
+maturity = "production"
+updated_date = "2020/12/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies PowerShell process loading the Task Scheduler COM DLL followed by an outbound RPC network connection within a
+short time period. This may indicate lateral movement or remote discovery via scheduled tasks.
```

Comment:
```suggestion
Identifies the PowerShell process loading the Task Scheduler COM DLL followed by an outbound RPC network connection within a
short time period. This may indicate lateral movement or remote discovery via scheduled tasks.
```

Reply:
may need to re-lint, otherwise looks good